code_generation/analysis_template.cxx

```diff
@@ -104,6 +106,9 @@ int main(int argc, char *argv[]) {
     // file logging
     Logger::enableFileLogging("logs/main.txt");
 
+    // start an onnx session manager
+    OnnxSessionManager onnxSessionManager;
```

Comment:
Why don't we need this in the friend template?

Reply:
fixed